### ISSUE TITLE
Central scarring of skin lesions

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#9962]
**Preferred term label:**
Central scarring of skin lesions

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Chromoblastomycosis

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
is this really characteristic? See https://www.ncbi.nlm.nih.gov/books/NBK470253/
I think this term as such is difficult because it is too closely tied to one diagnosis. Is there a general description of the phenotypic observation?

**Comment by MickeySegal** [comment]
Some other examples:
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9997686/ "Annular Basal Cell Carcinoma Expanding Around Central Hypertrophic Scarring: A Case Report"

https://www.dovepress.com/central-centrifugal-cicatricial-alopecia-challenges-and-solutions-peer-reviewed-fulltext-article-CCID
"Their hypothesis was derived from studying ten African American women with central scarring alopecia who did not engage in the hair practices noted by the previous authors (ie, straightening with a hot comb and petrolatum)."

It seems to be a term in wide use.

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001205
- label: Central scarring
- parents: [Spatial pattern (HP:0012836)](https://hpo.jax.org/app/browse/term/HP:0012836)
- pmid: PMID:36911588|PMID:29261968
- def: Central scarring of skin lesions.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9962
- synonyms:
